xml/System.Reflection.PortableExecutable/PEStreamOptions.xml

```diff
@@ -159,6 +168,19 @@
           <para>
             <see cref="T:System.Reflection.PortableExecutable.PEReader" /> closes the stream automatically by the time the constructor returns unless <see cref="F:System.Reflection.PortableExecutable.PEStreamOptions.LeaveOpen" /> is specified. The underlying file may be closed and even deleted after <see cref="T:System.Reflection.PortableExecutable.PEReader" /> is constructed.</para>
         </summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
```

Comment:
Part of this info is already in the summary. The rest should be moved to the class remarks if necessary,

Reply:
Removing.